### ISSUE TITLE
Modals using H1

### ISSUE THREAD

**Issue by leecollings-sb** [twbs/bootstrap#37244]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The latest version state this:

`Use <h1> for all .modal-title instances in our docs`

You shouldn't be doing this, this will lead to MULTIPLE H1s on the page markup, which is terrible for SEO and will instantly reduce your pagescore.

This shouldn't be in your docs, because this will lead many people down a path that will ruin their page rankings. This should be reverted back to a H3 or something H2 or lower.

### Reduced test cases

n/a

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

v5.2.2

**Comment by GeoSot** [comment]
Please try to be helpful next time, including the proper referrals in issue, avoiding spamming duplications 
It is as annoying as confusing to find the same comments in 3 places and on top without linking them.


Closing it as it is answered below
https://github.com/twbs/bootstrap/pull/37210#issuecomment-1265112629
https://github.com/twbs/bootstrap/issues/37179#issuecomment-1265114267

and continued on #37210

**Comment by leecollings-sb** [comment]
None of my issues are completed though, so this can't be closed.
